src/models/TalkMapper.php

```diff
@@ -772,4 +772,16 @@ public function delete($talk_id)
 
         return true;
     }
+
+    public function getSpeakerFromTalk($talk_id, $display_name)
+    {
+        $speaker_sql  = 'select ts.* from talk_speaker ts '
+            . 'where ts.talk_id = :talk_id and ts.speaker_name = :display_name';
+        $speaker_stmt = $this->_db->prepare($speaker_sql);
+        $speaker_stmt->execute(array("talk_id" => $talk_id, "display_name" => $display_name));
+        $speakers = $speaker_stmt->fetchAll(PDO::FETCH_ASSOC);
+
+        return count($speakers) > 0 ? $speakers[0] : false;
```

Comment:
I see where you are coming from, but I'm not sure that's the right approach in a mapper - I would always expect a getter in a mapper to return - either the data exists, or it doesn't.

Certainly we don't throw exceptions anywhere else in getters in the mappers - we do on set/create methods, and I think that makes sense, but this would be the 'odd one out' otherwise?